android/java/res/layout/fragment_backup_wallet.xml

```diff
@@ -66,8 +66,61 @@
             android:buttonTint="@color/brave_action_color"
             android:layout_marginStart="24dp"
             android:layout_marginEnd="24dp"
-            android:layout_marginBottom="40dp"/>
+            android:layout_marginBottom="30dp"/>
 
+        <TextView
+            android:id="@+id/tv_backup_wallet_password_title"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:gravity="center_horizontal"
+            android:layout_marginStart="32dp"
+            android:layout_marginEnd="32dp"
+            android:layout_marginBottom="24dp"
+            android:text="@string/backup_wallet_password"
+            android:textColor="@color/wallet_text_color"
+            android:visibility="gone"
+            android:textSize="14sp"/>
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="48dp"
+            android:layout_marginEnd="48dp"
+            android:layout_marginBottom="32dp"
+            android:gravity="center"
+            android:orientation="horizontal">
+
+            <EditText
+                android:id="@+id/et_backup_wallet_password"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_weight="1"
+                android:background="@drawable/rounded_wallet_edittext"
+                android:hint="@string/password"
+                android:importantForAutofill="no"
+                android:inputType="textPassword"
+                android:paddingStart="18dp"
+                android:paddingTop="10dp"
+                android:paddingEnd="18dp"
+                android:paddingBottom="10dp"
+                android:textColor="@color/wallet_text_color"
+                android:textColorHint="@color/brave_theme_hint_text"
+                android:textCursorDrawable="@null"
+                android:textSize="14sp"
+                android:visibility="gone" />
```

Comment:
nit: prefer to use style [BraveWalletEditTextBordered](https://github.com/brave/brave-core/blob/cfa565535dbb299c90375fc3253719354174eb79/android/java/res/layout/fragment_dapps_message.xml#L12)